app/__init__.py

```diff
@@ -26,12 +32,24 @@
 logging.basicConfig(level=logging.DEBUG)
 
 app = Quart(__name__)
-app = cors(app)
 
 for key in config.keys():
     app.config[key] = config[key]
 
+app = cors(app, allow_headers=['X-Requested-With'], allow_origin=app.config['ALLOW_ORIGIN'])
+
 load_config()
 
+if "pytest" in sys.modules:
+    from mockredis import mock_strict_redis_client
+    store = RedisStore(mock_strict_redis_client())
+else:
+    store = RedisStore(redis.StrictRedis(host=app.config['REDIS_HOST']))
```

Comment:
The redis store must be password protected. Currently it's very easy for a user to hack redis (in fact I tried...) 
![image](https://user-images.githubusercontent.com/13014091/47904002-c95a7c80-de84-11e8-9148-cc8d4ff1b4bb.png)

Reply:
Update: This is actually not as pressing as I thought because the network policy will prevent this kind of access for non-minikube deployments (as pointed out by @leafty).